packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -13,7 +13,7 @@
 // Use __VARIANT__ to simulate a GK. The tests will be run twice: once
 // with the __VARIANT__ set to `true`, and once set to `false`.
 
-export const alwaysThrottleRetries = true;
+export const alwaysThrottleRetries = __VARIANT__;
```

Comment:
Is this relevant?

Reply:
no I mistakenly reset this while debugging a different variant bug failing a test that was actually a problem in the test itself